.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Plone 6.1",
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.12-bullseye",
+
+	// Use 'portsAttributes' to set default properties for specific forwarded ports. 
+	// More info: https://containers.dev/implementors/json_reference/#port-attributes
+	"portsAttributes": {
+		"8080": {
+			"label": "Plone Backend",
+			"onAutoForward": "notify"
+		}
+	},
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "pip3 install --user -r requirements.txt"
```

Comment:
@xiru We should run `make install` here. This takes care of some other steps that build the requirements file to include some packages from github (for the packages that have unreleased changes).

We'll need a base image that includes tools like make, git, and ssh.